.github/workflows/examples.yml

```diff
@@ -1,22 +1,38 @@
+# NOTE: This workflow seems to do nothing but install the library through conda
```

Comment:
do we need this then? do we test somewhere else if it can be installed through conda? why is it called examples?

Reply:
I saw we test it in pytest.yml.
Remove this file?